apps/meteor/client/views/admin/oauthApps/EditOauthApp.tsx

```diff
@@ -47,12 +47,12 @@ const EditOauthApp = ({ onChange, data, ...props }: EditOauthAppProps): ReactEle
 	const authUrl = useMemo(() => absoluteUrl('oauth/authorize'), [absoluteUrl]);
 	const tokenUrl = useMemo(() => absoluteUrl('oauth/token'), [absoluteUrl]);
 
-	const saveApp = useMethod('updateOAuthApp');
-	const deleteApp = useMethod('deleteOAuthApp');
+	const saveApp = useEndpoint('POST', '/v1/oauth-apps.update');
+	const deleteApp = useEndpoint('DELETE', '/v1/oauth-apps.delete');
```

Comment:
```suggestion
	const deleteApp = useEndpoint('POST', '/v1/oauth-apps.delete');
```
If we decide to change the delete endpoint to POST, then this should be updated too.